README.md

```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Why would you use something like the `load` event? Does this event have disadvantages? Do you know any alternatives, and why would you use those?
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* What are some of the advantages/disadvantages of writing javascript code in a language that compiles to javascript?
+
+#### Testing and Performance Questions:
+
+* What are some advantages/disadvantages to testing your code?
+* What tools would you use to test your code's functionality?
+* What is the different between a unit test and a functional/integration test?
+* What tools would you use to find a performance bug in your code(CPU spike, memory leak, etc.)?
+* What are some ways you can improve your website's scroll performance if it is under 60 frames per second?
```

Comment:
I think you can be less specific here. No need to mention 60 frames per second. I think simply `What are some ways you may improve your website's scrolling performance?` would be sufficient